Framework/Algorithms/test/IntegrationTest.h

```diff
@@ -447,11 +451,11 @@ class IntegrationTest : public CxxTest::TestSuite {
     Workspace2D_sptr space2D = boost::dynamic_pointer_cast<Workspace2D>(space);
 
     for (int j = 0; j < 5; ++j) {
-      for (int k = 0; k < 5; ++k) {
-        space2D->dataX(j)[k] = 0.9 * k;
-        space2D->dataY(j)[k] = 2 * k + double(j);
-        space2D->dataE(j)[k] = 1.0;
-      }
+      auto &e = space2D->mutableE(j);
+      space2D->setPoints(j, Points(5, LinearGenerator(0, 0.9))); // assign X
+      space2D->setCounts(j,
+                         Counts(5, LinearGenerator(double(j), 2))); // assign Y
```

Comment:
You should not need the explicit constructor calls here, e.g., `space2D->setPoints(j, 5, LinearGenerator(0, 0.9));` should compile and do the same thing.